.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+package-lock.json
```

Comment:
I would prefer the package-lock.json to be removed from the .gitignore as it reduces errors by installing the exact same version of dependencies everytime and sometimes makes debugging easier.